pkgs/applications/networking/instant-messengers/telegram/tdesktop/default.nix

```diff
@@ -64,9 +64,7 @@
 
 let
   tg_owt = callPackage ./tg_owt.nix {
-    abseil-cpp = abseil-cpp.override {
-      cxxStandard = "17";
-    };
+    inherit abseil-cpp;
```

Comment:
I guess you can just remove that line?

Reply:
If we remove that line the override in all-packages.nix gets dropped.

Reply:
Ah, you're right